client/src/components/login/index.css

```diff
@@ -47,3 +47,15 @@
     padding-top: 1em;
     color: red;
 }
+
+/* ----------- MOBILE MEDIA QUERIES ----------- */
+
+@media screen and (max-width: 800px) {
```

Comment:
out of curiosity-- where is this `800px` value coming from?

Reply:
i read this article https://developers.google.com/web/fundamentals/design-and-ux/responsive/ where it says "create breakpoints based on content, never on specific devices, products, or brands." -- 800px seemed to make most of the pages look good when you shrunk it down that much? after that it looked fine wide, also we can always change it! i didn't want a million breakpoints for every possible phone tablet etc. it seemed like bloat.